.agent/workflows/lint.md

```
@@ -0,0 +1,180 @@
+
+#### Few-shot (how to flag issues deterministically)
+
+Use these patterns as a strict reference when scanning.
+
+1. Prefer transformations over mutable loops when it improves clarity:
+
+```kotlin
+// Smell: mutable accumulation with multiple branches
+val out = mutableListOf<Foo>()
+for (x in xs) {
+	if (x.isValid()) out += Foo(x)
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)